hl-todo.el

```diff
@@ -50,10 +50,10 @@
 
 ;;; Code:
 
-(require 'cl-lib)
 (require 'compat)
-
-(eval-when-compile (require 'subr-x))
+(eval-when-compile
+  (require 'subr-x)
+  (require 'cl-lib))
```

Comment:
I would prefer.

```elisp
(eval-when-compile (require 'subr-x))
(eval-when-compile (require 'cl-lib))
```

Because then, for each individual dependency, all the information is available on a single line.